### PR TITLE
use python module pipenv not system environment one

### PR DESCRIPTION
**Pull request by sahasrara62**
i am purposing using pipenv from the python modules not from the user sytem environment

### CODE REVIEW

**Comment by jamescurtin** [comment]
Thanks for the PR! Could you describe the problem this PR is trying to solve?

**Comment by sahasrara62** [comment]
@jamescurtin  i came across a point where in my system i have multiple python env and multiple packages with different version (all maintain in virtual env), i haven't installed the pipenv in system with direct python , i used env packages as `python -m <package> <rest of stuff>`  so when i run this code, i was not using pipenv as it suppose, so i had to change package.json according to my env configuration.

 so creating a PR to make a this little change, so script take default python (venv one) site packages packages not from the system packages

File: {{cookiecutter.app_name}}/requirements/dev.txt [hunk 1]
```diff
@@ -16,3 +16,4 @@ flake8-docstrings==1.5.0
 flake8-isort==2.7.0
 isort==4.3.21
 pep8-naming==0.9.1
+pipenv==2018.11.26
```
Comment (jamescurtin) [inline {{cookiecutter.app_name}}/requirements/dev.txt]: If a user chooses to use `pipenv` in their project, the [post gen hook](https://github.com/cookiecutter-flask/cookiecutter-flask/blob/1e34130ad536b3159ec3e70e75a8f9687f5f49a1/hooks/post_gen_project.py#L20) will delete `requirements.txt` and `requirements/*` from the generated project. As a result, I don't believe adding this requirement here will have the desired effect.

File: {{cookiecutter.app_name}}/requirements/prod.txt [hunk 2]
```diff
@@ -1,5 +1,8 @@
 # Everything needed in production
 
+# pipenv
+pipenv==2018.11.26
```
Comment (jamescurtin) [inline {{cookiecutter.app_name}}/requirements/prod.txt]: Same here

File: {{cookiecutter.app_name}}/README.md [hunk 3]
```diff
@@ -45,10 +45,10 @@ Run the following commands to bootstrap your environment if you are unable to ru
 ```bash
 cd {{cookiecutter.app_name}}
 {%- if cookiecutter.use_pipenv == "yes" %}
-pipenv install --dev
-pipenv shell
+python -m pipenv install --dev
+python -m pipenv shell
 {%- else %}
-pip install -r requirements/dev.txt
+python -m pip install -r requirements/dev.txt
```
Comment (jamescurtin) [inline {{cookiecutter.app_name}}/README.md]: I'm not sure this one is necessary. `virtualenv` environments come with `pip` available by default, so there shouldn't be any issues with the wrong version of pip being used.

File: {{cookiecutter.app_name}}/README.md [hunk 4]
```diff
@@ -62,9 +62,9 @@ Once you have installed your DBMS, run the following to create your app's
 database tables and perform the initial migration
 
 ```bash
-flask db init
-flask db migrate
-flask db upgrade
+python -m flask db init
+python -m flask db migrate
+python -m flask db upgrade
```
Comment (jamescurtin) [inline {{cookiecutter.app_name}}/README.md]: Assuming the user has followed the installation instructions above (_i.e._ either running `python -m pipenv install --dev` or `pip install -r requirements/dev.txt`), `flask` should be added into the `/bin` of the appropriate virtualenv. Therefore, running these commands should use the correct version of flask and the `python -m` is unnecessary. Have you found this to not be the case?

File: {{cookiecutter.app_name}}/README.md [hunk 5]
```diff
@@ -45,10 +45,10 @@ Run the following commands to bootstrap your environment if you are unable to ru
 ```bash
```
Comment (jamescurtin) [inline {{cookiecutter.app_name}}/README.md]: Maybe consider adding an optional block above the code example reminding pipenv users that it must be installed first. Possibly:

```
{%- if cookiecutter.use_pipenv == "yes" %}
Make sure to first [install pipenv](https://pipenv.readthedocs.io/en/stable/install/#installing-pipenv).
{%- endif %}
```

**Comment by jamescurtin** [comment]
Hi @sahasrara62 Hope you don't feel like the review comments meant you need to close the PR; Your suggestion makes sense for the places where pipenv is invoked!

**Comment by sahasrara62** [comment]
@jamescurtin  they are good suggestion and can be implemented. closing this PR as now for no reason.

**Pull request closed without merging**